### DISCUSSION TITLE
/hyndsight/dark-themes-for-writing/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#221] [Announcements]
Dark themes for writing 
 https://robjhyndman.com/hyndsight/dark-themes-for-writing/

**Comment by robjhyndman** [comment]
```header
Originally posted by David Stern on 2015-03-18 05:34:55
````
Most people find it much harder to read, I think.

**Comment by robjhyndman** [comment]
```header
Originally posted by Janko Thyson on 2015-03-18 08:31:25
````
I like dark themes a lot as well and after reading your post was quite
excited that RStudio now ships with Monokai. However, I can\'t seem to
find it in the latest Windows version (0.98.1103). Which version and OS
are you using?

**Reply by robjhyndman** [reply]
```header
Originally posted on 2015-03-18 10:17:10
````
I\'m using version 0.99.292 on Ubuntu. But themes have been available on
all OSs for ages. Look under Tools and Global Options.

**Reply by robjhyndman** [reply]
```header
Originally posted by Francesco S. on 2015-03-18 10:43:02
````
Hi Rob, I have checked in RStudio (Version 0.98.1102) under Global
Options -\&gt; Appearance but there isn\'t Monokai.

**Reply by robjhyndman** [reply]
```header
Originally posted by Matthias Olberz on 2015-03-18 12:05:06
````
Hi Everyone,\
for your information: the new dark themes are only available for the
preview-release builds:\
[http://www.rstudio.com/prod\...](http://www.rstudio.com/products/rstudio/download/preview/)

**Comment by robjhyndman** [comment]
```header
Originally posted by vzemlys on 2015-03-18 09:39:21
````
Strange, for me it is the opposite. Dark themes strain my eyes, hence I
have strong dislike for them.

**Comment by robjhyndman** [comment]
```header
Originally posted by John Hammersley on 2015-03-18 11:37:18
````
Even though I only use the default theme, there are a wide range of
different color schemes to choose from on Overleaf \-- see this post
[https://www.overleaf.com/bl\...](https://www.overleaf.com/blog/194-new-year-new-name-new-free-features#!themes &quot;https://www.overleaf.com/blog/194-new-year-new-name-new-free-features#!themes&quot;)
for more details.

(PS: I\'m one of the founders of Overleaf, so if you use it, any
feedback is appreciated, thanks!)

**Comment by robjhyndman** [comment]
```header
Originally posted by buggyfunbunny on 2015-03-18 12:33:54
````
Well, if it were really all that better, we\'d have figured out, over
the last few thousand years, how to print books thus. Doesn\'t seem to
be a problem. All of those folks with Kindles and Nooks and such would
have made the switch, too. I don\'t believe they have.

**Comment by robjhyndman** [comment]
```header
Originally posted by David Burton on 2015-03-18 13:36:13
````
I quite like dark themes in general stylistically, but I find that it\'s
much worse affected by reflections from windows, etc, as the light text
becomes hard to read - I\'ve had to switch to a light theme at my
current workplace to improve readability

**Comment by robjhyndman** [comment]
```header
Originally posted by Samohyl on 2015-03-18 23:33:01
````
Thanks this was really helpful

**Comment by robjhyndman** [comment]
```header
Originally posted by Xi'an on 2015-03-19 06:49:55
````
Thank you Rob, being as well a Dark Side supporter, now I have no reason
no to switch to R studio!

**Comment by robjhyndman** [comment]
```header
Originally posted by aeon on 2015-03-19 08:36:40
````
Thanks for sharing the profile. I very much appreciate that you put work
into that, and spared me to do it myself.

**Comment by robjhyndman** [comment]
```header
Originally posted by Andrew Jackson on 2015-03-19 15:54:33
````
I really like the dark theme in Rstudio, but one issue I find
frustrating is that when the mouse pointer turns into a vertical bar
style when hovered over a script or cmd window, it can be very difficult
to locate it on the screen.

**Reply by robjhyndman** [reply]
```header
Originally posted by Eduardo Tabacman on 2015-06-11 14:04:24
````
I agree 100%. Has anyone found a solution to this problem?

**Comment by robjhyndman** [comment]
```header
Originally posted by kn1g on 2015-03-22 15:39:23
````
Hi, I love dark themes. Started with Sublime. Working in RStudio all day
with a dark theme. But there is one thing that\'s really annoying. The
dark theme Support in RStudio is buggy. I googled all day long to find a
way to change the highlight color (e.g. if you search for words etc.).
Dark highlight on dark background\... RStudio needs to change this to
make the dark themes complete.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2015-03-23 05:40:55
````
Make a suggestion at
[https://support.rstudio.com\...](https://support.rstudio.com/hc/en-us?community_id=public &quot;https://support.rstudio.com/hc/en-us?community_id=public&quot;).

**Comment by robjhyndman** [comment]
```header
Originally posted by Al-Ahmadgaid Asaad on 2015-03-23 09:26:52
````
I wish RStudio team will add more configuration on the theme of the UI,
it\'s good to have dark themes on the editor and console. But it would
be best if we could change the overall theme of the UI, like that in
sublime, or in PyCharm there is a Darcula theme.

**Comment by robjhyndman** [comment]
```header
Originally posted by Erik on 2015-03-25 10:21:46
````
If you have a mac, then you can press command + option + ctrl + 8 to
invert the entire screen.

**Comment by robjhyndman** [comment]
```header
Originally posted by Evil Mammoth on 2016-12-04 21:58:26
````
Thank you, the TeXstudio theme is great!

**Comment by robjhyndman** [comment]
```header
Originally posted by Zhixun He on 2017-06-08 04:51:56
````
Thank you!

**Comment by robjhyndman** [comment]
```header
Originally posted by bhaismachine on 2017-10-09 21:46:30
````
I have been using your texstudio dark theme for almost two years now.
Thank you!

**Comment by robjhyndman** [comment]
```header
Originally posted by Ronie Uliana on 2017-11-04 14:01:27
````
Thank you for the dark theme on TexStudio!

**Comment by robjhyndman** [comment]
```header
Originally posted by Damiano G. Preatoni on 2017-11-15 13:54:21
````
Thank you for sharing, awesome themes both for TexStudio and RStudio!

**Comment by robjhyndman** [comment]
```header
Originally posted by Jefferson Ferreira-Ferreira on 2018-01-27 22:29:49
````
Thanks for this post. I feel the reading/writing much more comfortable
using this profile.

**Comment by robjhyndman** [comment]
```header
Originally posted by Megan Bailey on 2018-03-26 07:42:07
````
Dark themes are good for me, but I also think it depends on a person.
For some people it\'s difficult to read using them.

**Comment by robjhyndman** [comment]
```header
Originally posted by Linda Cox on 2018-12-21 10:09:06
````
I set up a dark theme everywhere (Youtube, Google etc). My eyes hearts
of the white color.

**Comment by callpri** [comment]
I prefer to use [Codelobster editor](https://codelobster.com)
